openwisp_radius/migrations/0044_convert_user_credentials_data.py

```
@@ -0,0 +1,33 @@
+# Generated by Django 5.2.5 on 2025-11-18 00:00
+
+import json
+
+from django.db import migrations
+
+
+def convert_user_credentials_data(apps, schema_editor):
```

Ruff (0.14.14)

[warning] Unused function argument: `schema_editor`

(ARG001)